### PR TITLE
adding default SEARCH_TAG in .env (this is overriten on build stage)

### PR DESCRIPTION
**Pull request by paulbrodner**
we need this change in order to execute it on our CI tool

### CODE REVIEW

**Comment by paulbrodner** [comment]
@tunaaksoy  If everything is ok - we can merge this 
See comment on [SEARCH-1064](https://issues.alfresco.com/jira/browse/SEARCH-1064?focusedCommentId=584008&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-584008)

File: packaging/src/docker/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+SEARCH_TAG=${project.version}
```
Comment (tunaaksoy) [inline packaging/src/docker/.env]: Why do we need this .env file?

File: packaging/src/docker/5.x/.env [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# pass the repository image:
+# image: docker-internal.alfresco.com/platform:5.2.1-SNAPSHOT
+ALFRESCO_TAG=5.2.1-SNAPSHOT
+SEARCH_TAG=${project.version}
```
Comment (tunaaksoy) [inline packaging/src/docker/5.x/.env]: This won't work once we tag images with 1.1.0.x. I'll create a ticket for that.

File: packaging/src/docker/5.x/.env [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# pass the repository image:
```
Comment (tunaaksoy) [inline packaging/src/docker/5.x/.env]: I think we should either add a generic comment or not at all.

**Comment by tunaaksoy** [comment]
Also we need to think about the image versions (see my comment in https://github.com/Alfresco/SearchServices/pull/44/files)

**Pull request merged**